resources/views/docs/1/the-basics/screens.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Screens
+order: 850
+---
+# Screens
+
+The `Screen` facade lets you get information about the screens currently connected to the computer.
+
+```php
+use Native\Laravel\Facades\Screen;
+```
+
+## Displays
+
+The `displays` method gives you an array of information about all the displays actually used.
+
+The Display object represents a physical display connected to the system. A fake display may exist on a headless system, or a display may correspond to a remote, virtual display. If you use an external display with your laptop screen closed, the internal screen of your laptop will not be part of the array.
+
+See [Display object](https://www.electronjs.org/docs/latest/api/structures/display) documentation.
+
+```php
+$screens = Screen::displays();
+```
+
+The screen bounds are the desktop area that the screen covers. The `x` and `y` values are the top-left corner of the screen relative to the primary display, and the `width` and `height` values are the width and height of the screen.
+
+Example to found out in what screen the window is located:
+
+```php
+function getCurrentScreen()
+{
+    $screens = Screen::displays();
+    $window = Window::current();
+
+    foreach ($screens as $screen) {
+        $bounds = $screen['bounds'];
+        if ($window->x >= $bounds['x'] && $window->x <= $bounds['x'] + $bounds['width'] && $window->y >= $bounds['y'] && $window->y <= $bounds['y'] + $bounds['height']) {
+            return $screen;
+        }
+    }
+
+    return null;
+}
+```
```

Comment:
I think this should be a helper included in NativePHP - feels like a pretty common problem to solve with a consistent solution that would be widely applicable.

Would also make a great PR 😉

Reply:
These sound great 👍🏼

Reply:
I will try to put another PR next week then!